CONTRIBUTING.md

```diff
@@ -1,107 +1,174 @@
 ## Contributing
 
-Pull requests for bug fixes are welcome, but before submitting new features or changes to current functionality [open an issue](https://github.com/open-telemetry/opentelemetry-auto-instr-java/issues/new)
-and discuss your ideas or propose the changes you wish to make. After a resolution is reached a PR can be submitted for review.
+Pull requests for bug fixes are welcome, but before submitting new features
+or changes to current functionality [open an
+issue](https://github.com/open-telemetry/opentelemetry-auto-instr-java/issues/new)
+and discuss your ideas or propose the changes you wish to make. After a
+resolution is reached a PR can be submitted for review.
 
-In order to build and test this whole repository you need JDK 11+.
-Some instrumentations and tests may put constraints on which java versions they support.
-See [Executing tests with specific java version](#Executing tests with specific java version) below.
+In order to build and test this whole repository you need JDK 11+. Some
```

Comment:
@flands I very like the suggestion of Dan Allen from https://www.youtube.com/watch?v=r6RXRi5pBXg about "Write one sentence per line". It makes diffs and merges easier and don't change rendered layout in any way.

Reply:
oh, yes, the specification repo has been battling with this decision: https://github.com/open-telemetry/opentelemetry-specification/pull/564. let's see what they end up implementing, and use the same? seems like they are leaning in the direction u mention